### PR TITLE
Spanish Translation Modals for Sketch Menu

### PR DESCRIPTION
**Pull request by oruburos**
I have verified that this pull request:

* [x] has no linting errors (`npm run lint`)
* [x] is from a uniquely-named feature branch and has been rebased on top of the latest master. (If I was asked to make more changes, I have made sure to rebase onto master then too)
* [x] is descriptively named

### CODE REVIEW

File: client/modules/IDE/components/NewFileForm.jsx [hunk 1]
```diff
@@ -30,14 +31,14 @@ class NewFileForm extends React.Component {
             className="new-file-form__name-input"
             id="name"
             type="text"
-            placeholder="Name"
+            placeholder={i18n.t('NewFileForm.Placeholder')}
```
Comment (andrewn) [inline client/modules/IDE/components/NewFileForm.jsx:34]: Same comments as in #1537. In components, let's try and use _withTranslation_. 

The `validate()` function is a good case for using the `i18next.t()` function directly but here it should be `this.props.t()`.

Comment (oruburos) [reply client/modules/IDE/components/NewFileForm.jsx]: @andrewn I made the changes requested. The problematic was also present in the Folder modal

**Comment by oruburos** [comment]
@andrewn I made the changes requested. The problem was also present in the Folder modal. Can you take a look, please?

**Comment by andrewn** [comment]
Looks great!

**Pull request merged**